src/DynamoCoreWpf/UI/Converters.cs

```diff
@@ -3423,6 +3423,24 @@ public object ConvertBack(object value, Type targetType, object parameter, Cultu
         }
     }
 
+    /// <summary>
+    /// Receive a GeometryScaleSize value and if it matches the parameter passed will return a brush with a specific color
+    /// </summary>
+    public class ScaleSizeBackgroundConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if (value.Equals(parameter))
```

Comment:
braces

Reply:
braces added in the next PR: https://github.com/DynamoDS/Dynamo/pull/13734